helm/apf/templates/ingress.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "apf.fullname" . -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ $fullName }}
+  labels:
+    {{- include "apf.labels" . | nindent 4 }}
+  annotations:
+    traefik.ingress.kubernetes.io/router.entrypoints: web, websecure
+    cert-manager.io/cluster-issuer: {{ .Values.certificate.issuername | quote }}
+spec:
+  {{- if .Values.ingress.tls }}
+  tls:
+    - secretName: {{ $fullName }}-cert
+  {{- end }}
+  rules:
+    {{- range .Values.ingress.hosts }}
+    - host: {{ .host | quote }}
+      http:
+        paths:
+        {{- range .paths }}
+        - path: {{ .path }}
+          pathType: Prefix
+          backend:
+            service:
+              name: {{ $fullName }}
```

Comment:
This is not k8s 1.18+ compatible

See https://kubernetes.io/blog/2020/04/02/improvements-to-the-ingress-api-in-kubernetes-1.18/#putting-it-all-together